components/SponsorsSection.js

```diff
@@ -57,7 +57,7 @@ export default function TicketsSection () {
 function ItemSponsor ({ src, type }) {
   return (
     <div className={`sponsor-item-${type}`}>
-      <img src={src} width='100%' />
+      <img alt='Photo' src={src} width='100%' />
```

Comment:
Adding `alt='Photo'` doesn’t help with making the web site more accessible. The increased score doesn’t give any benefit if solved this way.

The correct value for the `alt` attribute would be the name of the sponsor, e.g. ‘Thai Programmers Association’, ‘Sprint3r’, ‘Cleverse’, …